meshio/vtk/_vtk.py

```diff
@@ -616,8 +617,13 @@ def pad(array):
         logging.warning("VTK ASCII files are only meant for debugging.")
 
     with open_file(filename, "wb") as f:
+        if info is None:
+            info_line = "written by meshio v{}\n".format(__version__)
+        else:
+            info_line = "%s\n" % info[0]
```

Comment:
Assert that `len(jnfo) == 0` here.

Reply:
Also, don't use the `%`-style formatting.

Reply:
Zero-length `info` is now tested. Also, `info[0]` longer than 255 characters is truncated.

String formatting fixed.